### ISSUE TITLE
socket.io.js failed for function io.socket.get() in Firefox and Safari

### ISSUE THREAD

**Issue by bmustata** [balderdashy/sails#2788]
Hi,

We have a working sails.js 0.10.x application which is doing sockets calls get/post/pun and is working very well in Chrome but now at all in FireFox or Safari. Any ideas?

On FireFox and Safari we get the "`io.socket` connected successfully." message but no actual verbose message related to sockets can be seen on the server.

**Comment by cambrozie** [comment]
Is this still reproducing on 0.10.x version?

**Comment by tjwebb** [comment]
Can you post the full error output?

**Comment by CWyrtzen** [comment]
Is this still going on in the latest version?

**Comment by sailsbot** [comment]
Thanks for posting, @bmustata.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
